front-end/src/app/shared/components/inputs/additional-info-input/additional-info-input.component.ts

```diff
@@ -23,8 +23,8 @@ export class AdditionalInfoInputComponent extends BaseInputComponent implements
       );
     }
 
-    if (this.transaction?.transactionType?.memoTextPrefix) {
-      this.initPrefix(this.templateMap.text4000, this.transaction?.transactionType?.memoTextPrefix);
```

Comment:
If we lose memoTextPrefix, the MULTISTATE_INDEPENDENT_EXPENDITURE will have its transaction type setting lost. It looks like the text_prefix will need to account for a default value.

See here:
https://github.com/fecgov/fecfile-web-app/blob/develop/front-end/src/app/shared/models/transaction-type.model.ts#L85
https://github.com/fecgov/fecfile-web-app/blob/develop/front-end/src/app/shared/models/transaction-types/MULTISTATE_INDEPENDENT_EXPENDITURE.model.ts#L46-L47

Reply:
Ah, that's a really good catch!  I've fixed it, and I've tested that Multistate Independent Expenditures work with the fix.